x-pack/plugins/enterprise_search/public/applications/workplace_search/views/overview/workplace_search_gate.tsx

```diff
@@ -0,0 +1,751 @@
+            </p>
+          </EuiText>
+        </Fragment>
+      ),
+      inputDisplay: i18n.translate(
+        'xpack.enterpriseSearch.workplaceSearch.gateForm.superSelect.synonymns.inputDisplay',
+        {
+          defaultMessage: 'Synonyms',
+        }
+      ),
+      value: i18n.translate(
+        'xpack.enterpriseSearch.workplaceSearch.gateForm.superSelect.value.synonymns.value',
+        {
+          defaultMessage: 'synonyms',
+        }
+      ),
+    },
+    {
+      dropdownDisplay: (
+        <Fragment>
+          <strong>
+            {i18n.translate(
+              'xpack.enterpriseSearch.workplaceSearch.gateForm.superSelect.analytics.title',
+              {
+                defaultMessage: 'Analytics',
+              }
+            )}
+          </strong>
+          <EuiText size="s" color="subdued">
+            <p>
+              {i18n.translate(
+                'xpack.enterpriseSearch.workplaceSearch.gateForm.superSelect.analytics.description',
+                {
+                  defaultMessage: "Record and review users' interactions with search results",
+                }
+              )}
+            </p>
+          </EuiText>
+        </Fragment>
+      ),
+      inputDisplay: i18n.translate(
+        'xpack.enterpriseSearch.workplaceSearch.gateForm.superSelect.analytics.inputDisplay',
+        {
+          defaultMessage: 'Analytics',
+        }
+      ),
+      value: i18n.translate(
+        'xpack.enterpriseSearch.workplaceSearch.gateForm.superSelect.value.analytics.value',
+        {
+          defaultMessage: 'analytics',
+        }
+      ),
+    },
+    {
+      dropdownDisplay: (
+        <Fragment>
+          <strong>
+            {i18n.translate(
+              'xpack.enterpriseSearch.workplaceSearch.gateForm.superSelect.other.title',
+              {
+                defaultMessage: 'Other',
+              }
+            )}
+          </strong>
+          <EuiText size="s" color="subdued">
+            <p>
+              {i18n.translate(
+                'xpack.enterpriseSearch.workplaceSearch.gateForm.superSelect.other.description',
+                {
+                  defaultMessage: 'Another feature not listed here',
+                }
+              )}
+            </p>
+          </EuiText>
+        </Fragment>
+      ),
+      inputDisplay: i18n.translate(
+        'xpack.enterpriseSearch.workplaceSearch.gateForm.superSelect.other.inputDisplay',
+        {
+          defaultMessage: 'Other',
+        }
+      ),
+      value: i18n.translate(
+        'xpack.enterpriseSearch.workplaceSearch.gateForm.superSelect.value.other.value',
+        {
+          defaultMessage: 'other',
+        }
+      ),
+    },
+  ];
+
+  const { setFormSubmitted, setAdditionalFeedback, setParticipateInUXLabs, setFeature } =
+    useActions(WorkplaceSearchGateLogic);
+
+  const { feature, participateInUXLabs } = useValues(WorkplaceSearchGateLogic);
+
+  return (
+    <WorkplaceSearchGatePageTemplate
```

Comment:
I would split this out to a separate page, probably merge it with the WorkplaceSearchGatePageTemplate you defined elsewhere. That's only used in one spot, so it doesn't really need a separate template. And this file is now over 700 lines long, which makes it hard to parse and keep track of.

Reply:
> That's only used in one spot, so it doesn't really need a separate template. 

Makes sense, will remove the `WorkplaceSearchGatePageTemplate` file and move to [workplace_search_gate.tsx](https://github.com/elastic/kibana/pull/167315/files/7e01bc02fa5bece4770373bc745f857ed2541381#diff-131b0bfa7b537fb37c8a9fa32e4da95314d4a8f4c2cee13d3b5fb5d59cc6965e).